ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -494,19 +486,16 @@ const ControlledTierEraModule = ({profile, active, spinnerProps}): JSX.Element =
   const Module = ({profile}) => {
     return <FlexRow data-test-id={`module-${moduleName}`}>
       <FlexRow style={styles.moduleCTA}>
-        {((showRefresh && refreshAction)
-            ? <Refresh
-                refreshAction={refreshAction}
-                showSpinner={spinnerProps.showSpinner}/>
-            : null)}
+        {showRefresh && refreshAction
+          && <Refresh refreshAction={refreshAction} showSpinner={spinnerProps.showSpinner}/>}
       </FlexRow>
       <ModuleBox>
-        <ModuleIcon moduleName={moduleName} completedOrBypassed={!!statusTextMaybe}/>
+        <ModuleIcon moduleName={moduleName} completedOrBypassed={isDone(status)}/>
         <FlexColumn>
-          <div style={!statusTextMaybe ? styles.activeModuleText : styles.inactiveModuleText}>
+          <div style={isDone(status) ? styles.inactiveModuleText : styles.activeModuleText}>
```

Comment:
interesting... this implies to me that the names of these styles are not quite right.  The distinction we're looking for is complete (background styling) vs incomplete (foreground styling).  Only one of the incomplete modules is also active.

Reply:
Agreed.